src/main/java/org/venice/beachfront/bfapi/config/GeoServerBaseAuthenticationConfig.java

```diff
@@ -0,0 +1,39 @@
+package org.venice.beachfront.bfapi.config;
+
+import org.apache.http.HeaderElement;
+import org.apache.http.HeaderElementIterator;
+import org.apache.http.HttpResponse;
+import org.apache.http.conn.ConnectionKeepAliveStrategy;
+import org.apache.http.message.BasicHeaderElementIterator;
+import org.apache.http.protocol.HTTP;
+import org.apache.http.protocol.HttpContext;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class GeoServerBaseAuthenticationConfig {
+
+	@Value("${http.keep.alive.duration.seconds}")
+	private int httpKeepAliveDurationSeconds;
+
+	/**
+	 * Defines a keep-alive strategy for requests that do not provide a proper timeout value. This will avoid a case
+	 * where keep-alive is set to permanent (when the header is missing) in cases where this application's environment
+	 * will not allow for permanent connections.
+	 */
+	protected ConnectionKeepAliveStrategy getKeepAliveStrategy() {
```

Comment:
This keepalive strategy is universal for all calls in BF-API, including the Piazza calls. As such, I don't think it belongs here in a class called GeoServer. This should be applied in all classes - could there be somewhere more generic to keep it?

Reply:
The`httpClient() ` and `restTemplate()` beans defined in the GeoServer* files apply for the entire springboot application. I think a good solution is to rename the files to be more BF-API generic. Do you concur?

Reply:
I'm fine with renaming the files to be more generic, yeah.

Reply:
Done.